dev-docs/maintenance.md

```diff
@@ -0,0 +1,140 @@
+# Maintaining Automate
+
+The automate code base needs continual maintenance to remain robust and fresh.
+
+Who owns this task? YOU.
+Every engineer should own this; taking the initiative to either fix something or at least report something to those who can, when issues arise.
+
+There are two main aspects to this:
+
+- Proactive: keeping up with third-party library releases
+- Reactive: addressing issues that come up due to system changes (which may include third-party libraries as well as changes in environments and so forth).
+
+## Third-party Dependency Update Monitoring
+
+Proactively one should monitor releases of third-party libraries and re-sync our code base with them as soon as scheduling allows.
+Automate uses two main languages, Go and TypeScript, each having their own complex ecosystem.
+
+### Backend Updates
+
+There is no central place to check on all dependencies.
+The following, though, lists some of the key Golang dependencies. One can set up a "watch" on each of these repositories to be automatically notified of new releases that can then be scheduled as technical debt items to be done in a timely fashion.
+
+- https://github.com/golang/go/releases
+- https://github.com/grpc/grpc-go/releases
+- https://github.com/golang/protobuf/releases
+- https://github.com/open-policy-agent/opa/releases
+- https://github.com/grpc-ecosystem/grpc-gateway/releases
+- https://github.com/golang/protobuf/protoc-gen-go/releases
+- https://github.com/stretchr/testify/releases
+- https://github.com/lib/pq/releases
+- core/dex
+- core/nginx
```

Comment:
Why are these listed as hab core packages vs. the other ones that have GitHub URLs here?

Reply:
Only because I did not know immediately where to find those references. 🤷

Reply:
Fixed.